.github/workflows/net-version-sweeper.yml

```diff
@@ -0,0 +1,42 @@
+# The name used in the GitHub UI for the workflow
+name: '.Net Version Sweeper'
+
+# When to run this action:
+# - Scheduled on the first of every month
+# - Manually runable from the GitHub UI with a reason
+on:
```

Comment:
Is it intentional that branch name is not mentioned?

Reply:
Yeah it's a monthly cron job so runs monthly not on pr, seemed to make sence. Do you think it should run on pr's @avnes ?